Sources/SwiftPhoenixClient/TimeoutTimer.swift

```diff
@@ -76,12 +76,13 @@ class TimeoutTimer {
     
     // Get the next calculated interval, in milliseconds. Do not
     // start the timer if the interval is returned as nil.
-    guard let timeInterval
-      = self.timerCalculation.call(self.tries + 1) else { return }
+      guard let timeInterval = self.timerCalculation?(self.tries + 1) else {
+          return
+      }
```

Comment:
Nit. There appears to be some kind of inconsistent indentation here.

I know the project uses 2 spaces and 4 spaces is more the standard. Something I plan on correcting later on when I add SwiftLint but for now let's stay consistent.

Reply:
I changed it back.

It's just a couple files that are indent at 2, right? Would you take a PR that standardized those? You can tell git/github to ignore the commit so it doesn't break blame/history (.git-blame-ignore-revs)